linkerd/docs/protocol-h2.md

```diff
@@ -402,13 +402,31 @@ kind: `io.l5d.default`.
 The default trace propagator stores the trace id in the `l5d-ctx-trace` request header.  It also
 reads the `l5d-sample` and, if present, uses this value as the sample rate for this request.
 
-#### Trace Propagator Configuration:
+<aside class="notice">
+The trace information in the header are serialized by Finagles `TraceId.serializeIt` method.
```

Comment:
I can't find the TraceId.serializeIt method.  Am I missing something?

Reply:
Ah, sorry thats a typo. ^^;;; 
The "Id" part was added by mistake, its TraceId.serialize.

See the Finagle doc: 
https://twitter.github.io/finagle/docs/com/twitter/finagle/tracing/TraceId$.html